application/helpers/common_helper.php

```diff
@@ -5105,3 +5105,23 @@ function isAbsolutePath($path)
         return false;
     }
 }
+
+/**
+ * Escapes a string for use in a CSV file
+ * @param string|null $string
+ * @return string
+ */
+function csvEscape($string)
```

Comment:
No need `preg_replace(array('~\R~u'), array("\n"), $value);` and `'"' . str_replace('"', '""', $sString) . '"';`

?

See https://github.com/LimeSurvey/LimeSurvey/blob/0cc10f52a6127da9f8267e658728af4c72a27cac/application/helpers/admin/export/CsvWriter.php#L102